### ISSUE TITLE
Changing terrain page size will mess up the map reliably

### ISSUE THREAD

**Issue by NetSysFire** [worldforge/ember#40]
![image](https://user-images.githubusercontent.com/59517351/206918164-4dc4d4e0-1999-407a-8c3a-0acc7de44eb3.png)

Steps to reproduce:

1. Connect to any server.
2. Change terrain page size.
3. Chaos.

Expected behavior:

Do not apply this setting until ember has restarted. Display a warning like "This setting will take effect after reconnecting to the server or restarting ember". Additionally, the terrain settings are missing tooltips.

**Comment by erikogenvik** [comment]
Yeah, this is an issue. Changing the terrain settings is mainly a developer setting, meant to try to find the best size. The terrain system as it is should however be replaced with one using Compute Shaders, so I'm not sure how much effort we want to put into fixing this issue.

**Comment by NetSysFire** [comment]
Feel free to assign this one low priority as it is mostly an edge case, unlikely to hit ordinary users. But if I recall correctly, this is not marked as a debug setting, so maybe moving it from the graphics into the troubleshooting section in the settings would also work.
